e2e/storageState.json

```diff
@@ -0,0 +1,15 @@
+{
+  "cookies": [
+    {
+      "name": "JSESSIONID",
+      "value": "3338D7F677DE81FAFD5CF0A0EBC1D972",
+      "domain": "localhost",
+      "path": "/openmrs",
+      "expires": -1,
+      "httpOnly": true,
+      "secure": true,
+      "sameSite": "Strict"
+    }
+  ],
+  "origins": []
+}
```

Comment:
Add EOF
```suggestion
}

```

Reply:
This file should be ignored

Reply:
@jayasanka-sack Should I remove it then

Reply:
Yes. Remove it and add it to .gitigone file. There are some other file that you should ignore. Please check the patient management repo. :)